Documentation/Books/Manual/Upgrading/Manually/Cluster.md

```diff
@@ -0,0 +1,129 @@
+--> TMP <---
```

Comment:
Anything planned here? If you want to add invisible comments (not rendered to HTML at all), you can use:

    {### some comment ###}